test/e2e/storage/vsphere/nodemapper.go

```diff
@@ -33,9 +33,11 @@ import (
 	neturl "net/url"
 )
 
+// NodeMapper holds method for node info map
 type NodeMapper struct {
 }
```

Comment:
So this is probably not a golint error, but this type stores its state in global variables `nameToNodeInfo` and `vcToZoneDatastoresMap`. Those should really be inside this struct. Let me know if you think that should be done in a separate PR.

Reply:
I think this should be done in a separate PR